react-class-component/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import flagsmith from 'flagsmith'
+    
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      fontSizeEnabled: false,
+      fontSize: 0,
+      identity: null,
+      trait: undefined,
+    }
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.identity !== this.state.identity) {
+      this.updateFlags()
+    }
+  }
+
+  updateFlags() {
+    try {
+      const fontSizeEnabled =  flagsmith.hasFeature('font_size');
+      const fontSize =  flagsmith.getValue('font_size');
+      const exampleTrait =  flagsmith.getTrait('example_trait');
+      this.setState({
+        fontSizeEnabled,
+        fontSize,
+        trait: exampleTrait,
+      })
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  handleLogout = async () => {
+    await flagsmith.logout()
+     this.updateFlags()
+    this.setState({ identity: undefined })
+  }
+
+  handleIdentify = async () => {
+    const identity = await flagsmith.identify('flagsmith_sample_user')
+     this.updateFlags()
+    this.setState({ identity })
+  }
+
+  render() {
+    const { fontSizeEnabled, fontSize, trait, identity } = this.state
+
+    return (
+      <div>
+        <h1>Flagsmith Example</h1>
+        <div className='App'>
+          font_size: {fontSize} example_trait: {trait}
```

Comment:
Should we display `fontSizeEnabled` somehow? Also would be cool to actually render the content with different font size.

Reply:
Good idea, I'll implement it before merging it.